packages/app/src/debug/empty/DebugEmptyStates.cy.tsx

```diff
@@ -119,12 +119,10 @@ describe('Debug page empty states', { defaultCommandTimeout: 250 }, () => {
     })
   })
 
-  context('error', () => {
+  context('not using git', () => {
     it('renders', () => {
       mountWithGql(<DebugError />)
 
-      cy.findByRole('link', { name: 'Learn more about debugging CI failures in Cypress' }).should('have.attr', 'href', 'https://on.cypress.io/debug-page?utm_source=Binary%3A+Launchpad&utm_medium=Debug+Tab&utm_campaign=Learn+More')
```

Comment:
Is this link no longer present on this view? Or is this just not a worthwhile assertion?

Reply:
It is now removed, I did add an assertion to validate the new copy displays in 28d8f3521ff8a7226481387a742309faade5ba60